python-avd/tox.ini

```diff
@@ -30,17 +30,17 @@ deps =
     pytest
     PyYAML>=6.0.0
     pydantic>=2.3.0
-    coverage
+    coverage[toml]
 commands =
     coverage erase
     coverage run -m pytest
 
 [testenv:report]
 skip_install = true
 deps =
-    coverage
+    coverage[toml]
 commands =
-    coverage report
+    coverage report --rcfile=pyproject.toml
 
 [coverage:report]
 fail_under = 85
```

Comment:
Please update to the new number. Also this should probably move to the rest of the config settings?